app/src/main/java/com/goryachok/forecastapp/WeatherApplication.kt

```diff
@@ -2,13 +2,17 @@ package com.goryachok.forecastapp
 
 import android.app.Application
 import android.content.Context
+import com.goryachok.forecastapp.dagger.splashViewModel.DaggerViewModelComponent
+import com.goryachok.forecastapp.dagger.splashViewModel.ViewModelComponent
 import com.goryachok.forecastapp.network.api.OpenWeatherMapAPI
 import com.goryachok.forecastapp.network.connectivity.ConnectivityInterceptor
 import com.goryachok.forecastapp.network.data.PREFS_NAME
 import com.goryachok.forecastapp.network.data.WeatherDataRepository
 
 class WeatherApplication : Application() {
```

Comment:
Please, create interface App() and inherit Application class from it. Extract 'component' as interface member.

Reply:
Done